incuna-sass/_mixins.sass

```diff
@@ -3,3 +3,8 @@
 @import mixins/placeholder
 @import mixins/rgba-background
 @import mixins/svg-background
+@import mixins/svg
+// If your project does not have sprites folders set up
+// the following may result in an error when you compile
+// your sass
+@import mixins/sprites
```

Comment:
Once imported, cannot be un-imported, so a project won't be able to say "no, I don't want sprites".